### ISSUE TITLE
Tray icons out of place when connecting external monitor

### ISSUE THREAD

**Issue by PhdTrollSlayer** [jaagr/polybar#691]
So, I connect my monitor and use this command to turn him on:
`xrandr --output HDMI-1 --mode 1360x768 --left-of eDP-1 --rotate right`

And the tray icons look like this 
https://imgur.com/a/uOCBL

Any ideas on how to fix it??

**Comment by patrick96** [comment]
I don't see what's wrong with that. How does it look when the monitor is not connected?
Can you also post your config

**Comment by PhdTrollSlayer** [comment]
Config:

https://gist.github.com/PhdTrollSlayer/147e8211311eae67551d18fa27c80347

**Comment by PhdTrollSlayer** [comment]
http://imgur.com/1YXqNFM

**Comment by patrick96** [comment]
Does that happen too, when you start the bar, while the monitor is already connected and configured?
Can you start the bar with the `-l info` argument, once normally and then connect the monitor and once with the monitor already connected and configured and post the logs of both

**Comment by PhdTrollSlayer** [comment]
Ok, I think I fixed. I'm using a script that I made to connect the monitor:
https://gist.github.com/PhdTrollSlayer/8e3d18e9e20a9bffc6fe315ba94143e7
Also I set it up polybar to open only on eDP-1 (My laptop screen).
Working fine now

**Comment by patrick96** [comment]
Great!

**Comment by bin4ryio** [comment]
Hello, didn't mean to threadjack, but my polybar has the same issue (tray out of position, 100% same) using default config. Should I open up a new issue?

I tried setting `monitor = ${env:MONITOR:eDP-1}` does not help. I've tried with/without `wm-restack`, `override-redirect`. I also tried starting `polybar default -l info` it does not show any error other than a `XEMBED_WINDOW` and bunch of unknown glyph.
